utils/virtcontainers-setup.sh

```diff
@@ -59,7 +59,7 @@ cp -r ${busybox_bundle}/* ${pause_bundle}/
 echo -e "Move to ${tmpdir}/${virtcontainers_build_dir}"
 pushd ${tmpdir}/${virtcontainers_build_dir}
 echo "Clone cni"
-git clone https://github.com/containernetworking/cni.git
+git clone https://github.com/containernetworking/plugins.git
```

Comment:
Frankly, I'm still not happy with this, however I see now that this change is "no worse" than what was there before (which was also just pulling from the tip). So in the interests of unbreaking the build let's merge this. However, I think we need to do better going forward and consider whether we should just pull the bleeding edge here.